src/mode/doc_comment_highlight_rules.js

```diff
@@ -5,16 +5,97 @@ var TextHighlightRules = require("./text_highlight_rules").TextHighlightRules;
 
 var DocCommentHighlightRules = function() {
     this.$rules = {
-        "start" : [ {
-            token : "comment.doc.tag",
-            regex : "@[\\w\\d_]+" // TODO: fix email addresses
-        }, 
+        "start": [
+            {
+                token: ["comment.doc.tag", "text", "lparen.doc"],
```

Comment:
should this be in generic doc_comment highlight rules or specific to js and ts?

Reply:
You are right, this seems should be specific to js and ts